src/containers/pages/InterventionPlan/IRS/index.tsx

```diff
@@ -169,12 +176,19 @@ class IrsPlans extends React.Component<IrsPlansProps & RouteComponentProps<Route
           <title>{IRS_TITLE}</title>
         </Helmet>
         <HeaderBreadcrumbs {...breadCrumbProps} />
-        <h2 className="page-title">{IRS_PLANS}</h2>
+        <Container>
+          <Row>
+            <Col md="8">
+              <h2 className="page-title">{IRS_PLANS}</h2>
+            </Col>
+            <Col md="4" className="text-right">
+              <Button color="primary" tag={Link} to={`${INTERVENTION_IRS_URL}/new`}>
+                Create new plan
+              </Button>
+            </Col>
+          </Row>
+        </Container>
         <DrillDownTable {...tableProps} />
```

Comment:
Let's wrap the DrillDownTable in a Row/Col in the new Container to align things.

<img width="1193" alt="Screen Shot 2019-09-19 at 8 16 23 AM" src="https://user-images.githubusercontent.com/4141527/65257470-1869ca00-dab6-11e9-8be2-6196dab006d5.png">

Reply:
Done